test/setupTests.ts

```diff
@@ -4,3 +4,16 @@
  */
 
 import 'babel-polyfill';
+import fetch, { Request, Response } from 'node-fetch';
```

Comment:
How about we separate a `fetch-polyfill` file and import here? Just like the node-fetch recommendation https://github.com/node-fetch/node-fetch#providing-global-access